### DISCUSSION TITLE
New Honeypots Coming to T-Pot

### DISCUSSION THREAD

**Discussion by t3chn0m4g3** [telekom-security/tpotce#1629] [Announcements]
We are working hard behind the scenes to identify and integrate new honeypots for the next T-Pot release, which will be T-Pot 24.04.1 (planned for 🎃🎄'ish).

If everything works out as planned (one or maybe even two or more) LLM based honeypots will be integrated as well.

This is thanks to all the dedicated developers I had the chance to talk to at this year's Honeynet Workshop, here on GitHub and my personal network.

And of course Telekom Security by sponsoring the necessary hardware to get this started!

Should you plan any PRs make sure to discuss them in advance 👋

Not on T-Pot 24.04 yet? Time to get started!

**Comment by dmille6** [comment]
thank you for all that you do @t3chn0m4g3

**Comment by mariocandela** [comment]
LLM based honeypots 🚀 Bye-bye high-interaction honeypot.

**Reply by t3chn0m4g3** [reply]
Exactly what I am thinking 🤩

**Comment by t3chn0m4g3** [comment]
T-Pot 24.04.1 - LLM honeypot integration sneak peek...

![image](https://github.com/user-attachments/assets/8d4cb26d-f1fb-47d6-b540-b5dc057e6613)

**Reply by dmille6** [reply]
i'm very interested in this, thank you for your continued work on this project.

**Reply by mariocandela** [reply]
awesome! :rocket:

**Reply by t3chn0m4g3** [reply]
Almost done ...
![image](https://github.com/user-attachments/assets/415fc87f-bba3-46fa-92fc-9b165aee2842)

**Reply by t3chn0m4g3** [reply]
Kibana 8.16.1 viz are much better for viewing on large displays ...
![image](https://github.com/user-attachments/assets/f2294aa4-a823-48af-84be-9580b2a42f32)

**Reply by dmille6** [reply]
thank you for all your hard work

**Reply by t3chn0m4g3** [reply]
... and released #1713!
